orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/kubernetes/KubernetesJobRunner.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2019 Netflix, Inc.
```

Comment:
s/Netflix/Armory/

Reply:
🤦‍♂

Reply:
Oh, this was a Groovy -> Java conversion so I just kept the original header.